File: app/src/main/java/com/lin/read/activity/LaunchActivity.java
package com.lin.read.activity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.lin.read.R;

import java.util.Timer;
import java.util.TimerTask;

/**
 * Created by lisonglin on 2017/10/11.
 */

public class LaunchActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_launch);
    }

    @Override
    protected void onResume() {
        super.onResume();

        TimerTask task = new TimerTask() {
            @Override
            public void run() {
                startActivity(new Intent(LaunchActivity.this, MainActivity.class));
                finish();
            }
        };
        Timer timer = new Timer();
        timer.schedule(task, 3000);
    }
}

File: app/src/main/java/com/lin/read/activity/LaunchActivity.kt
package com.lin.read.activity

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.os.Handler
import com.lin.read.R

class LaunchActivity : Activity(){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_launch)
    }

    override fun onResume() {
        super.onResume()
        Handler().postDelayed({
            startActivity(Intent(this@LaunchActivity, MainActivity::class.java))
            finish()
        }, 200)
    }
}